darts/utils/timeseries_generation.py

```diff
@@ -308,6 +308,65 @@ def random_walk_timeseries(mean: float = 0.,
     return TimeSeries.from_times_and_values(index, values, freq=freq, columns=pd.Index([column_name]))
 
 
+def autoregressive_timeseries(coef: Sequence[float],
+                              start_values: Optional[Sequence[float]] = None,
+                              start: Optional[Union[pd.Timestamp, int]] = pd.Timestamp('2000-01-01'),
+                              end: Optional[Union[pd.Timestamp, int]] = None,
+                              length: Optional[int] = None,
+                              freq: str = 'D',
+                              column_name: Optional[str] = 'autoregressive') -> TimeSeries:
+    """
+    Creates a univariate, autoregressive TimeSeries whose values are calculated using specified coefficients `coef` and
+    starting values `start_values`.
+
+    Parameters
+    ----------
+    coef : list of float
```

Comment:
```suggestion
    coef : Sequence of float
```

Reply:
I also don't like 'list' but it is what the numpydoc styleguide suggests [numpydoc - styleguide - parameters](https://numpydoc.readthedocs.io/en/latest/format.html#:~:text=4.-,Parameters,-%C2%B6)
Should I still change it to 'Sequence' ?

Reply:
I didn't know that! Let's leave `list` then.